README.md

```diff
@@ -1797,6 +1797,19 @@ P.S that's Riziy and he help me study ;)
 
 ![Riziy kitten](assets/images/riziy-vcheryasova.jpg)
 
+---
+
+#Hello kottans!
```

Comment:
```suggestion
# Hello kottans!
```
If you want it to be heading level 1, you need add space here.

Reply:
Thanks for advice! Edited it

Reply:
Oh, just realized that I could commit your suggestion instead of editing file by myself)